requirements/docs.txt

```diff
@@ -14,4 +14,5 @@ pooch>=1.3.0
 tifffile>=2020.5.30
 myst-parser
 ipywidgets
-plotly>=4.10.0
+plotly>=4.14.0
+kaleido
```

Comment:
I wasn't familiar with `kaleido`, but it looks like plotly uses it for static image export when it is available.

Reply:
Right, see https://plotly.com/python/static-image-export/.